### PR TITLE
fix: update base image reference in script

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

**What type of PR is this?**

/king bug

**What this PR does / why we need it**:
- Updates the base image reference in the build file used for scripts.

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,7 +30,7 @@ E2E_PROVIDER_IMAGE_NAME ?= e2e-provider
 # Release version is the current supported release for the driver
 # Update this version when the helm chart is being updated for release
 RELEASE_VERSION := v1.2.3
-IMAGE_VERSION ?= v1.2.3
+IMAGE_VERSION ?= v1.2.3.0
```
Comment (nilekhc) [inline Makefile:33]: Why are we adding an additional 0 at the end?

Comment (aramase) [reply Makefile:33]: Any change to `docker/` dir in release branch will trigger a new build. We don't want to rebuild `v1.2.3` release again, so this is dummy value.

**Pull request merged**